command/healthcheck/healthcheck.go

```diff
@@ -110,12 +110,12 @@ func (e *Executor) Execute() (map[string][]*Result, error) {
 		}
 
 		if err := checker.FetchResources(e); err != nil {
-			return nil, err
+			return nil, fmt.Errorf("failed to execute %v: %w", checker.Name(), err)
```

Comment:
```suggestion
			return nil, fmt.Errorf("failed to fetch resources %v: %w", checker.Name(), err)
```

Reply:
I kinda wanted to leave this a little vague for the caller -- they don't really care if it fails in the fetch stage or early in the evaluation stage I think. Both are fundamentally the same step of the health check, to them, its only in implementation that we make that distinction I feel?

Let me know if you disagree :-)

Reply:
Yup that is fine as well, it was only to differentiate if we get a generic error back from an end-user we could zero in a little more on what step actually had failed.